src/iac_scan_runner/project_config.py

```diff
@@ -0,0 +1,135 @@
+        except Exception as e:
+            print(f"Could not parse JSON, error: {e}")
+            
+    def insert_config(self, result: dict):
+        """Inserts new config into database
+        :param result: Dictionary holding the project info
+        """     
+        if self.connection_problem:
+            self.connect_db()
+        if self.mycol is not None:
+            self.mycol.insert_one(self.parse_json(result))
+
+    def new_config(self, creatorid: str, parameters: dict) -> str:
+        """Inserts new project config into database
+        :param creatorid: Identifier of project creator
+        :param result: Identifier of a new configuration
+        """     
+        config_json = dict()
+        
+        # TODO: 	Add this names into object
+        config_json["configid"] = str(uuid.uuid4())
+        config_json["creatorid"] = creatorid
+                      
+        if(parameters):
+            config_json["parameters"] = dict()
+            config_json["parameters"] = parameters            
+        else:
+            config_json["parameters"] = dict()
+            
+        if self.connection_problem:
+            self.connect_db()
+        if self.mycol is not None:
+            self.mycol.insert_one(self.parse_json(config_json))
+            return config_json["configid"]
+         
+    def load_config(self, configid: str) -> dict: 
+        """Shows scan project with given id
+        :param projectid: Identifier of a scan project
+        :return: JSON object of project
+        """
+        if self.connection_problem:
+            self.connect_db()
+            
+        if self.mycol is not None:                
+            myquery = {"configid": configid}
+            mydoc = self.mycol.find(myquery)
+            for x in mydoc:
+                return x
+
+    def set_parameters(self, configid: str, parameters: dict):    
+        """Changes an active configuration of a given scan project
+        :param projectid: Identifier of a scan project
+        :param configid: Identifier of currently active project configuration that we want to assign     
+        :return: JSON object of user
+        """
+
+        if self.connection_problem:
+            self.connect_db()
+
+        if(self.mycol != None):           
+
+            myquery = {"configid": configid}
+            new_value = {"$set": {"parameters": parameters}}
+            try:
+                self.mycol.find_one_and_update(myquery, new_value, upsert=True)
+            except Exception as e:
+                print(f"Could not update project configuration {configid} error: {e}")                
+    
+    def delete_config(self, configid: str):    
+        """Deletes the configuration with given id from database
+        :param configid: Identifier of a project configuration which is about to be deleted
+        """
+        if self.connection_problem:
+            self.connect_db()
+            
+        if(self.mycol is not None):          
```

Comment:
There are still a lot of these kind of formatting errors. Please go through the code and refactor them all.

```suggestion
        if self.mycol is not None:          
```

Reply:
Fixed